src/api/import.js

```diff
@@ -0,0 +1,205 @@
+import debug from 'debug';
+import fs from 'fs-promise';
+import inquirer from 'inquirer';
+import path from 'path';
+import { spawn as yarnOrNPMSpawn, hasYarn } from 'yarn-or-npm';
+
+import initGit from '../init/init-git';
+import { deps, devDeps } from '../init/init-npm';
+
+import asyncOra from '../util/ora-handler';
+import installDepList from '../util/install-dependencies';
+import readPackageJSON from '../util/read-package-json';
+import confirmIfInteractive from '../util/confirm-if-interactive';
+
+const d = debug('electron-forge:import');
+
+/**
+ * @typedef {Object} ImportOptions
+ * @property {string} [dir=process.cwd()] The path to the app to be imported
+ * @property {boolean} [interactive=false] Whether to use sensible defaults or prompt the user visually
+ */
+
+/**
+ * Attempt to import a given module directory to the Electron Forge standard.
+ *
+ * - Replaces the prebuilt electron package with the one that integrates with `electron-compile`
+ * - Sets up `git` and the correct NPM dependencies
+ * - Adds a template forge config to `package.json`
+ *
+ * @param {ImportOptions} providedOptions - Options for the import method
+ * @return {Promise} Will resolve when the import process is complete
+ */
+export default async (providedOptions = {}) => {
```

Comment:
Can we give these functions Regular Names? I would be surprised if ESDoc generates readable docs with this